ledger/catchpointfilewriter_test.go

```diff
@@ -854,7 +854,7 @@ func TestExactAccountChunk(t *testing.T) {
 	catchpointFilePath := filepath.Join(tempDir, t.Name()+".catchpoint.tar.gz")
 
 	cph := testWriteCatchpoint(t, dl.validator.trackerDB(), catchpointDataFilePath, catchpointFilePath, 0)
-	require.EqualValues(t, cph.TotalChunks, 1)
+	require.EqualValues(t, cph.TotalChunks, 2)
```

Comment:
why is there one more chunk? Added online accounts overgrew the max chunk size?

Reply:
@cce can you speak to this?